stable/prometheus-pushgateway/templates/deployment.yaml

```diff
@@ -23,13 +23,13 @@ spec:
         - name: pushgateway
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+          {{- range $key, $value := .Values.extraArgs }}
+            - --{{ $key }}={{ $value }}
```

Comment:
this breaks existing behaviour

Reply:
Hey @davidkarlsen,

thanks for looking into it and finding this.

This time I installed the default prometheus-pushgateway with the default values as it is and then used my fork to upgrade with disks / persistence.

Here I installed the prometheus-pushgateway as it is
`helm3 install prometheus-pushgateway-test -f charts_upstream_master/stable/prometheus-pushgateway/values.yaml --namespace prometheus-pushgateway charts_upstream_master/stable/prometheus-pushgateway`

Here I used my fork with pvc support and my adjusted values.yaml
`helm3 upgrade prometheus-pushgateway-test -f prometheus-pushgateway-test/helm_chart_values-custom-chart.yaml --namespace prometheus-pushgateway charts/stable/prometheus-pushgateway`

I checked that I could reach the pushgateway and verified the configuration for persistence was changed / added.

This worked fine. If you have another test I should do please let me know.